Danggn/Screen/DetailItem/Controller/DetailItemViewController.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  DetailItemViewController.swift
+//  Danggn
+//
+//  Created by 황찬미 on 2022/05/19.
+//
+
+import UIKit
+
+class DetailItemViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+    }
+
+    @IBAction func homeButton(_ sender: Any) {
```

Comment:
이 메소드를 호출하는게 UIButton이니까 sender를 UIButton으로 명시적으로 표시해주는게 어떨까요~?🧐
```suggestion
    @IBAction func homeButton(_ sender: UIButton) {
```

Reply:
수정하였습니다 ~

Reply:
우리 회의 때 정한 컨벤션대로 메소드 명을  `homeButtonDidTap`으로 가는 것도 좋을 것 같아요!
```suggestion
    @IBAction func homeButtonDidTap(_ sender: UIButton) {
```

Reply:
엇 이거를 놓쳤네요 지금 수정하겠습니다

Reply:
이걸 늦게 봐서 ㅠ_ㅠ ... 다시 커밋 날렸습니다